### DISCUSSION TITLE
Backoffice: form modal readonlyOnView

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#180] [Q&A]
Hi

I have a backoffice application with a `bk-form-modal` having the following structure:
```
{
                "type": "element",
                "tag": "bk-form-modal",
                "properties": {
                    "dataSchema": {
                        "$ref": "dataSchema"
                    },
                    
                    "readonlyOnView": true,
                    "editorHeight": 250
                }
            },
```
However, if `readonlyOnView` is set to `true`, `editorHeight` is not applied to the editor height and is set to default. How can I solve this issue?

Thanks

**Comment by amountainram** [comment]
Hey there, would you mind reporting which `back-kit` lib version you're using? Thanks

**Reply by flower-of-the-bridges** [reply]
Hi @amountainram ! I'm using the latest version of back-kit

**Comment by clabene** [comment]
Hi @giovf96, config seems alright. Would you please [fill an issue](https://makeitapp.atlassian.net/wiki/spaces/BIB/pages/3706454026/How+to+open+a+Backoffice+issue+or+request+a+new+feature)? We will analyze and get back to you as soon as possible.
